docs/releases.md

```diff
@@ -175,7 +175,7 @@ Become part of the Tor Project and turn your DietPi into a Tor relay to help oth
 
 For more details check the [documentation page](../software/advanced_networking/#tor-relay).
 
-![advanced-networking-tor](assets/images/dietpi-software-advanced-networking-tor.png){: width="200" height="121" loading="lazy"}
+![advanced-networking-tor](assets/images/dietpi-software-distributed-projects-tor.png){: width="200" height="121" loading="lazy"}
```

Comment:
It does not hurt, but I don't see much reason to strictly rename those image file names. Especially Tor remains to be an advanced networking software as well, when used with Tor HotSpot. Generally IMO these long image names are unnecessary and sometimes misleading when the same logos or screenshots are used in multiple locations. `tor.png` or `tor-logo.png` would be just fine.